packages/test-runner/test/fixtures.spec.ts

```diff
@@ -0,0 +1,151 @@
+/**
+ * Copyright Microsoft Corporation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { expect } from '@playwright/test-runner';
+import { fixtures } from './fixtures';
+const { it } = fixtures;
+
+it('should work', async ({runInlineTest}) => {
+  const {results} = await runInlineTest({
+    'a.test.js': `
+      fixtures.defineTestFixture('asdf', async ({}, test) => await test(123));
+      it('should use asdf', async ({asdf}) => {
+        expect(asdf).toBe(123);
+      });
+    `,
+  });
+  expect(results[0].status).toBe('passed');
+});
+
+
+it.skip('should work with a non-async function', async ({runInlineTest}) => {
```

Comment:
Let's never use skip, mark it as failing.

Reply:
What is the api to do that? I only had .only and .skip

Reply:
Check out how we do that in playwright.